imageprocessor/migrations/0001_initial.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.1.1 on 2018-10-21 01:16
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
Do you have a feel for how this will affect our memory issues? I've really only have a cursory understanding of how migrations and storage works in django.